run_cgw.sh

```diff
@@ -17,37 +17,43 @@ DEFAULT_DB_PORT=5432
 DEFAULT_REDIS_DB_IP="127.0.0.1"
 DEFAULT_REDIS_DB_PORT=6379
 
+LOG_LEVEL="${CGW_LOG_LEVEL:-info}"
 ID="${CGW_ID:-1}"
-
-GRPC_IP="${CGW_GRPC_IP:-$DEFAULT_SRC_IP}"
-GRPC_PORT="${CGW_GRPC_PORT:-$DEFAULT_GRPC_PORT}"
 WSS_LOCAL_IP="${CGW_WSS_IP:-$DEFAULT_WSS_IP}"
 WSS_LOCAL_PORT="${CGW_WSS_PORT:-$DEFAULT_WSS_PORT}"
+GRPC_IP="${CGW_GRPC_IP:-$DEFAULT_SRC_IP}"
+GRPC_PORT="${CGW_GRPC_PORT:-$DEFAULT_GRPC_PORT}"
 KAFKA_IP="${CGW_KAFKA_IP:-$DEFAULT_KAFKA_IP}"
 KAFKA_PORT="${CGW_KAFKA_PORT:-$DEFAULT_KAFKA_PORT}"
 DB_IP="${CGW_DB_IP:-$DEFAULT_DB_IP}"
 DB_PORT="${CGW_DB_PORT:-$DEFAULT_DB_PORT}"
-DB_USR="${CGW_DB_USER:-cgw}"
-DB_PASS="${CGW_DB_PASS:-123}"
+DB_USERNAME="${CGW_DB_USER:-cgw}"
```

Comment:
My recommendation is for all of these variables you plan to pass to the Docker container, prefix the line with `export` such as:
```
export LOG_LEVEL="${CGW_LOG_LEVEL:-info}"
...
export DB_USERNAME="${CGW_DB_USER:-cgw}"
...
```

And then when calling docker run, give the variable name in `-e` but don't give the value, such as:
```
docker run \
	-e CGW_LOG_LEVEL        \
...
	-e CGW_DB_USERNAME \
```
This way the values are never passed on the command line, and are always inherited through the environment variables.